systest/tests/smeshing_test.go

```diff
@@ -15,7 +15,7 @@ import (
 
 func TestSmeshing(t *testing.T) {
 	tctx := testcontext.New(t, testcontext.Labels("sanity"))
-	cl, err := cluster.Default(tctx, cluster.WithKeys(10))
+	cl, err := cluster.Reuse(tctx, cluster.WithKeys(10))
```

Comment:
how does this affect test running locally?

Reply:
it shouldnt, unless you want to reuse cluster